Forms/Main.cs

```diff
@@ -2422,7 +2429,7 @@ private void InitDownloadList()
             this.ListDownloads.Clear();
             this.ListDownloads.HideAllProgress();
 
-            const string DefaultColSizes = "0,2.49|1,0.81|2,1.28|3,1.04|4,0.6";
+            const string DefaultColSizes = "0,2.49|1,0.81|2,1.28|3,1.04|4,0.6|5,1.0";
```

Comment:
I think that this could probably do with being a bit bigger - on my machine it only shows up as `Programme N...`

`1.2` gets the whole title to display, but perhaps it should be `1.3` or `1.4` to display more of the programme names.